docs/additional-examples/evm-swap.md

```diff
@@ -0,0 +1,656 @@
+# Cross Chain EVM Swap
+In this section, we will demonstrate how you can use PKPs and Lit Actions to perform a cross-chain swap. There are three parts to this flow:
+
+- Understanding Lit PKPs as escrow
+- Lit Action
```

Comment:
> Lit Action

can we be more specific here, like we were with the PKP usage? What are Lit Actions being used for in this flow? Let's highlight that

Reply:
Maybe something like "using Lit Actions to define the swap parameters" ?